src/System.Private.ServiceModel/src/System.Private.ServiceModel.csproj

```diff
@@ -59,4 +57,4 @@
     <TargetingPackReference Include="Windows" />
   </ItemGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
-</Project>
+</Project>
```

Comment:
Visual Studio made these changes automatically. It's some simple formatting tidy up. If it's checked in, it means other people don't need to keep ignoring that VS made changes.